index.rst

```diff
@@ -245,6 +245,97 @@ It is also worth mentioning again that the overall throughput we achieve with Ka
+   
+   Support for CI will still be minimum in jenkins, however since this is supposed to be a drop-in implementation we do not expect any roadblocks.
+   
+   We expect that ts_cycle_build will also be branched out to support building the system for Kafka.
+
+   Because ts_idl will no longer be necessary, it will probably be marked for deprecation at this stage.
+
+#. CI fully supported in jenkins.
+
+   At this point we expect that running CI jobs in jenkins will be fully supported.
+
+   The main focus is to support running jobs for salobj and SAL kafka branches.
+   We also should have in place the release jobs, that will make the SAL libraries available for C++ and Java.
+
+   For Python/salobj, there is no additional support needed as conda packaging will be able to provide all the required artifacts.
+
+#. Initial tests at the summit.
+
+   Once we have certified that we can run the system reliably on TTS, we should schedule testing at the summit.
+   The idea is to deploy the same system that was tested on TTS at the summit and execute a 3-5 nights long AuxTel run as well as any ongoing tests with the SST.
+   If possible it would be desirable to run both telescopes at the same time, driven by the Scheduler.
+   However, we must be attentive to the summit calendar and Kafka testing should be done in a way to minimize impact.
+
+   For this step, the kafka versions of both ts_salobj and ts_sal will be on a separate branch and we should rely on release candidate versions for deployment.
+
+   Development will still target the DDS version of salobj and sal.
+
+#. Adoption evaluation.
+
+   After testing at the summit concludes we will evaluate the results and produce a technote with our findings
+
+   If the test succeeds we can keep the summit running the Kafka version of our stack.
+
+#. Developer migration.
+
+   At this point TSSW developers are expected to migrate from the DDS version of salobj and SAL to the kafka version.
+
+   All development shall target the kafka version of salobj and SAL.
+   At this time we expect developers might have to take some time fixing potential issues with unit tests to run their components against the kafka version of the tssw-stack.
+
+   We do not expect code changes to be necessary to adapt to kafka.
+   However, if necessary, this will be the time to do those changes.
+
+#. Initial transition to Kafka.
+
+   At this point all development and deployment will have been migrated to target the kafka version of tssw stack.
+   
+   We will start making a census of the breaking changes that are planned for a full migration to kafka.
+   For example, the ts-idl package will be deprecated and all enumerations will migrate to ts-xml.
+   Developers will start incorporating these changes to their software as time permits.
+
+#. Final transition to Kafka.
+
+   Release of version 8 of salobj implementing breaking changes.
+
```

Comment:
* What if the breaking changes require code changes to CSCs? Do we need to spell out what happens?
* Should we give a timeline for the endgame on `ts-idl`?